kaolin/datasets/shapenet.py

```diff
@@ -193,6 +160,85 @@ def __getitem__(self, index):
         return {'data': data, 'attributes': attributes}
 
 
+class ShapeNet(KaolinDataset):
+    r"""ShapeNet Dataset class for meshes.
+    Args:
+        root (str): path to ShapeNet root directory
```

Comment:
We should maybe specify the version of ShapeNet we're expecting

Reply:
ShapeNetV1 ?

Reply:
Yeah, and also see my other comment on `model.obj` and `model_normalized.obj`

Reply:
Done